ui/components/multichain/app-header/app-header.scss

```diff
@@ -0,0 +1,52 @@
+.multichain-app-header {
+  flex-flow: column nowrap;
+  z-index: 55;
+  min-height: 64px;
+
+  &__contents {
+    display: grid;
+    grid-template-columns: 1fr 2fr 1fr;
+    height: 64px;
+
+    @include screen-sm-max {
+      height: 100%;
+    }
+
+    @include screen-sm-min {
+      width: 85vw;
+    }
+
+    @include screen-md-min {
+      width: 80vw;
+    }
+
+    @include screen-lg-min {
+      width: 62vw;
+    }
+  }
+
+  &__lock-contents {
+    flex-flow: row nowrap;
+    height: 64px;
+
+    @include screen-sm-max {
+      height: 100%;
+    }
+
+    @include screen-sm-min {
+      width: 85vw;
+    }
+
+    @include screen-md-min {
+      width: 80vw;
+    }
+
+    @include screen-lg-min {
+      width: 62vw;
```

Comment:
Should these values be stored in CSS variables if they are being reused in this stylesheet?

Reply:
Since we are going to create a container component for this, I think we can update it with that?

Reply:
I meant you could create some css variables within this stylesheet seeing as these values are duplicated

Reply:
Updated here https://github.com/MetaMask/metamask-extension/pull/18363/commits/b2d9ac4a8d7bf325395b5b0c768482b50b35406d